watertap/unit_models/zero_order/low_pressure_pump_zo.py

```diff
@@ -0,0 +1,71 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+"""
+This module contains a zero-order representation of a low pressure pump unit
+"""
+
+from pyomo.environ import Constraint, Reference, units as pyunits, Var
+from idaes.core import declare_process_block_class
+
+from watertap.core import build_pt, pump_electricity, ZeroOrderBaseData
+
+# Some more information about this module
+__author__ = "Chenyu Wang"
+
+
+@declare_process_block_class("LowPressurePumpZO")
+class LowPressurePumpZOData(ZeroOrderBaseData):
+    """
+    Zero-Order model for a low pressure pump unit operation.
+    """
+
+    CONFIG = ZeroOrderBaseData.CONFIG()
+
+    def build(self):
+        super().build()
+
+        self._tech_type = "low_pressure_pump"
+
+        build_pt(self)
+
+        self.lift_height = Var(units=pyunits.feet,
+                            doc="Lift height for pump")
+
+        self.eta_pump = Var(units=pyunits.dimensionless,
+                            doc="Efficiency of pump")
+
+        self.eta_motor = Var(units=pyunits.dimensionless,
+                            doc="Efficiency of motor")
+
+        self.electricity = Var(
+            self.flowsheet().config.time,
+            units=pyunits.kW,
+            bounds=(0, None),
+            doc="Electricity for low pressure pump")
+
+        self._fixed_perf_vars.append(self.lift_height)
+        self._fixed_perf_vars.append(self.eta_pump)
+        self._fixed_perf_vars.append(self.eta_motor)
+
+        @self.Constraint(self.flowsheet().time,
+                         doc='Constraint for electricity consumption based on '
+                             'pump flowrate.')
+        def electricity_consumption(b, t):
+            q_in = pyunits.convert(b.properties[t].flow_vol,
+                                   to_units=pyunits.gallon / pyunits.minute)
+            A = 3960 * pyunits.gallon * pyunits.foot / pyunits.minute / pyunits.horsepower
+            return b.electricity[t] == pyunits.convert(
+                q_in * self.lift_height / (A * self.eta_pump * self.eta_motor),
```

Comment:
here, to calculate pump energy consumption using SI units, you would multiply `lift_height` in meters by density of solution and gravitational acceleration. Density is taken from the property model, and you can use `acceleration_gravity`  `from idaes.core.util.constants import Constants` to handle gravitational acceleration.